### DISCUSSION TITLE
Can we self-host renovate backend

### DISCUSSION THREAD

**Discussion by mtanida** [renovatebot/renovate#21073] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Gitlab Runner

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hi! We were exploring using Renovate on our Gitlab projects using the Gitlab Runner, but we were surprised to learn that you still need a Github token and that renovate send API requests to some external service. Our codebase isn't open source and the InfoSec team has security concerns about this.

Is there a way that we can self-host the backend services so that renovate doesn't need to send requests to any external service? 

Thank you!!!!!

### What you tried so far.

Successfully ran on dummy project, but security concerns prevent us from using it on our actual projects.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Renovate doesn't "phone home". See https://github.com/renovatebot/renovate/discussions/19553 for details

It does need to reach out to servers like github, npm, etc if you want anything to be updated

**Reply by mtanida** [reply]
Thanks, Rhys. It seemed to be sending a request to some GraphQL API so that's where the question comes from. I will share this information with our InfoSec team.

**Reply by rarkins** [reply]
That's likely to look up github releases or release notes.

**Reply by mtanida** [reply]
Thank you
